manifests/d/Daum/PotPlayer/220106/Daum.PotPlayer.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Daum.PotPlayer
+PackageVersion: "220106"
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- 3gp
+- avi
+- mkv
+- mov
+- mp4
+- ogm
+Installers:
```

Comment:
We already have a waiver for https://t1.daumcdn.net which is the official CDN of Global PotPlayer

https://t1.daumcdn.net/potplayer/PotPlayer/Version/220106/PotPlayerSetup64.exe
https://t1.daumcdn.net/potplayer/PotPlayer/Version/220106/PotPlayerSetup.exe

Reply:
Oh, i got this one from Chocolatey, my bad